### PR TITLE
Make autobump job use an image with ssh

### PR DESCRIPTION
**Pull request by fejta**
We need ssh to push the git change to github.

Also:
* Configure name and email for the commit (extra args to autopush)
* Mount in the ssh secret
* Delete TestBazelbuildArgs, no one uses it this way any more

/assign @BenTheElder @krzyzacy

### CODE REVIEW

File: config/jobs/kubernetes/test-infra/test-infra-trusted.yaml [hunk 1]
```diff
@@ -103,16 +103,23 @@ periodics:
     base_ref: master
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/gcloud-bazel:v20181107-d88dbbf
+    - image: gcr.io/k8s-testimages/bazelbuild:v20181117-55a41ad84-0.18.1
```
Comment (stevekuznetsov) [inline config/jobs/kubernetes/test-infra/test-infra-trusted.yaml]: Could always use kubekins for everything :)

**Comment by krzyzacy** [comment]
lol obviously we made assumptions for jobs that uses a bazelbuild image

**Comment by fejta** [comment]
PTAL!!

**Pull request merged**